src/applications/fry-dea/components/ConfirmationResponses.jsx

```diff
@@ -152,6 +154,116 @@ UnderReview.prototype = {
   response: PropTypes.object || PropTypes.bool,
 };
 
+export function Approved(name) {
+  return (
+    <>
+      <div>
+        <va-alert
+          close-btn-aria-label="Close notification"
+          status="success"
+          visible
+        >
+          <h2 slot="headline">
+            Congratulations, you’ve been approved for Survivors’ and Dependents’
+            Educational Assistance
+          </h2>
+          <div>
+            <p className="vads-u-margin-y--0 vads-u-margin-bottom--3">
+              We reviewed your application and have determined that you’re
+              entitled to educational benefits under Survivors’ and Dependents’
+              Educational Assistance (DEA, Chapter 35). Your decision letter is
+              now available. A physical copy will also be mailed to your mailing
+              address.
+            </p>
+            <a
+              href="https://www.va.gov"
+              rel="noopener noreferrer"
+              target="_blank"
+            >
+              <i
+                className="fa fa-download vads-u-margin-right--1"
+                aria-hidden="true"
+              />
+              <span className="vads-u-margin-left--0p25 vads-u-font-weight--bold">
+                Download your decision letter (PDF)
+              </span>
+              .
```

Comment:
Please move this period outside of the anchor element.

Reply:
resolved